test/Configure/is_conftest/is_conftest.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+#
+# __COPYRIGHT__
```

Comment:
Please use the new copyright blurb..   (you can see it in `template/test.py`)

Reply:
updated to the template copyright